common/metrics/defs.go

```diff
@@ -648,6 +648,8 @@ const (
 	FrontendRespondActivityTaskCanceledByIDScope
 	// FrontendGetWorkflowExecutionHistoryScope is the metric scope for frontend.GetWorkflowExecutionHistory
 	FrontendGetWorkflowExecutionHistoryScope
+	// FrontendGetRawHistoryScope is the metric scope for frontend.GetWorkflowExecutionHistory
+	FrontendGetRawHistoryScope
```

Comment:
The test will fail if you only add the def here.

Also I think you need add another metrics for `FrontendClientGetRawHistoryScope`